src/jsonpatcherproxy.js

```diff
@@ -193,10 +211,13 @@ const JSONPatcherProxy = (function() {
     * @constructor
     */
   function JSONPatcherProxy(root, showDetachedWarning) {
+    /**
+     * Use tree[this._metadataSymbol] to access the tree metadata when all you have is a reference to a proxified version of the tree.
+     */
+    this._metadataSymbol = Symbol("Symbol for getting the tree metadata from external access.");
```

Comment:
Why it's a new symbol per instance? Can'w we create one Symbol for all instances?
Is it to be able to use JSONProxyPatcher on same object twice, and get notified twice?

Reply:
It is to be able to use JSONPatcherProxy twice on the same object, and be sure that each instance can keep its own metadata for its traps.

However, there is no test for such usage.

Reply:
I think we need such test, to prove this change is needed and make sure the functionality will survive ;)